utils/sse_calls.h

```diff
@@ -41,11 +41,12 @@ extern "C" {
 #define SSE_TRUNCATE_FLOAT_TO_INT(X)     _mm_cvttps_epi32(X)
 #define SSE_SQUARE_FLOAT(X)              _mm_mul_ps(X,X)
 #define SSE_SET_FLOAT(X)                 _mm_set1_ps(X)
+#define SSE_CONVERT_INT_TO_FLOAT(X)      _mm_cvtepi32_ps(X)
```

Comment:
Same as the AVX512F case - you might need a "free" cast operation rather than the convert operation